packages/menu-bar/src/vaadin-lit-menu-bar-overlay.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright (c) 2016 - 2023 Vaadin Ltd.
```

Comment:
Should be aligned with the other new files

Reply:
Done.